src/BrefServiceProvider.php

```diff
@@ -58,6 +60,13 @@ public function register()
         Config::set('queue.connections.sqs.secret');
         Config::set('queue.connections.sqs.token', env('AWS_SESSION_TOKEN'));
         Config::set('queue.connections.sqs.prefix', env('SQS_PREFIX', "https://sqs.{$region}.amazonaws.com/{$account}"));
+
+        $this->app->when(QueueHandler::class)
+            ->needs('$connection')
+            ->giveConfig('queue.default');
+        $this->app->when(QueueHandler::class)
+            ->needs('$queue')
+            ->giveConfig('queue.connections.sqs.queue');
```

Comment:
Here I pre-configure the `QueueHandler` service so that it can be created by the Laravel container directly, without any extra instructions.

Reply:
How does one run multiple queues? Say I have queue-a and queue-b, how do I configure 2 lambda functions that both point to their respective queues?

Reply:
Yeah, it's fairly common to have `high`, `default` and `low` queues.

Reply:
Good point! I pushed a new commit that actually helps resolving that problem (unused parameter).

What that latest commit, how about using this config as a user:

```yaml
provider:
    ...
    environment:
        # default queue
        QUEUE_CONNECTION: sqs
        SQS_QUEUE: ${construct:jobs.queueUrl}

constructs:

  jobs:
    type: queue
    worker:
      handler: CacheWerk\BrefLaravelBridge\Queue\QueueHandler
      runtime: php-81

  emails:
    type: queue
    worker:
      handler: CacheWerk\BrefLaravelBridge\Queue\QueueHandler
      runtime: php-81
      environment:
        QUEUE_CONNECTION: emails
```

In the second queue worker we override the connection via the env variable.

Is that what you had in mind?

Reply:
Other than overriding `SQS_QUEUE` instead of `QUEUE_CONNECTION`, does exactly what I meant yes.

Bottom line is that both connection and queue can be overridden using environment variables.

Reply:
As in one queue worker might use SQS and emails and the other one Redis and default.

Reply:
👍